WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -44,9 +46,9 @@ public WalletViewModel(UiContext uiContext, IWalletModel walletModel, Wallet wal
 		_title = WalletName;
 
 		// TODO:
-		//this.WhenAnyValue(x => x.IsCoinJoining)
-		//	.Skip(1)
-		//	.Subscribe(_ => MainViewModel.Instance.InvalidateIsCoinJoinActive());
+		////this.WhenAnyValue(x => x.IsCoinJoining)
+		////	.Skip(1)
+		////	.Subscribe(_ => MainViewModel.Instance.InvalidateIsCoinJoinActive());
```

Comment:
Is this really what `CodeFactor` suggest to us?
Double `//` without any space after them?
Looks weird to me.

Reply:
> How to Fix Violations
To fix a violation of this rule, ensure that the comment begins with a single space. If the comment is being used to comment out a line of code, ensure that the comment begins with four forward slashes, in which case the leading space can be omitted.

https://documentation.help/StyleCop/SA1005.html

Reply:
Alright, thanks for the explanation.